instruqt/delivering-as-an-appliance/03-collecting-configuration/assignment.md

```diff
@@ -11,16 +11,355 @@ notes:
   contents: |
     Help your users configure your application with a simple form
 tabs:
-- title: Shell
+- id: h7hyb0imphye
+  title: Release Editor
+  type: code
+  hostname: shell
+  path: /home/replicant
+- id: tvyvcofb2ocv
+  title: Shell
   type: terminal
   hostname: shell
 difficulty: basic
 timelimit: 300
 ---
 
-Configuring your application for their own environment can create
-a challenge for your customers. Helm values are great, but
-documenting a giant values file and distringuishing which values to
-change is a challenge. The configuration capabilities of the
-Replicated Embedded Cluster let you offer a simple web form to assure
-your customers set critical values correctly.
+One of the benefits of building a Kubernetes appliance is providing a guided
+experience for your customer to have a successful installation. One key to
+success is assuring the application is correctly configured. The Replicated
+Embedded Cluster lets you offer a simple web screen to help customers set
+critical values correctly.
+
+Configuring Your Application
+=============================
+
+Like other aspects of the appliance, the configuration screen is defined using
+a YAML manifest. The manifest defines a `Config` resource that describes you
+configuration options and the screen that your customer will use to configure
+the application. Configurations is organized into groups, and each group can
+have a fields of different types. The type of field defines the widget used.
+The supported types account for all of the standard web form inputs.
+
+<table>
+<thead>
+<tr><th>Type</th><th>Widget</th></tr>
+</thead>
+<tbody>
+<tr><td><code>bool></code></td><td>checkbox</td></tr>
+<tr><td><code>file></code></td><td>file upload</td></tr>
+<tr><td><code>heading></code></td><td>a subheading within a group</td></tr>
+<tr><td><code>label></code></td><td>an label supporting another field</td></tr>
+<tr><td><code>password></code></td><td>password field with obscured text and visibility
+toggle</td></tr>
+<tr><td><code>select_one></code></td><td>radio buttons</td></tr>
```

Comment:
```suggestion
<tr><td><code>radio</code></td><td>radio buttons</td></tr>
```
select one is deprecated in favor of radio (also there's a new dropdown item type)

Reply:
not when I wrote this 😉